### PR TITLE
AQTS-632-FI-free-text-validation

### PR DESCRIPTION
**Pull request by syed87**
I've updated the error message and hit text on the page where there is a free text response to and FI request

https://dfedigital.atlassian.net/jira/software/projects/AQTS/boards/207?selectedIssue=AQTS-632

### CODE REVIEW

File: app/forms/teacher_interface/further_information_request_item_text_form.rb [hunk 1]
```diff
@@ -6,7 +6,10 @@ class FurtherInformationRequestItemTextForm < BaseForm
     attribute :response, :string
 
     validates :further_information_request_item, presence: true
-    validates :response, presence: true
+    validates :response,
+              presence: {
+                message: "Please respond to the assessor's request",
+              }
```
Comment (Hassanmir92) [inline app/forms/teacher_interface/further_information_request_item_text_form.rb:12]: We should use the `I18n` locals instead.

Take a look at `teacher_interface.en.yml` file for other examples.

File: spec/forms/teacher_interface/further_information_request_item_text_form_spec.rb [hunk 2]
```diff
@@ -14,7 +14,6 @@
   let(:response) { nil }
 
   it { is_expected.to validate_presence_of(:further_information_request_item) }
-  it { is_expected.to validate_presence_of(:response) }
```
Comment (Hassanmir92) [inline spec/forms/teacher_interface/further_information_request_item_text_form_spec.rb:17]: Why is this removed?

File: app/forms/teacher_interface/further_information_request_item_text_form.rb (hunk 1 above)
Comment (Hassanmir92) [reply app/forms/teacher_interface/further_information_request_item_text_form.rb:12]: You shouldn't have to make any changes here.

File: spec/forms/teacher_interface/further_information_request_item_text_form_spec.rb (hunk 2 above)
Comment (syed87) [reply spec/forms/teacher_interface/further_information_request_item_text_form_spec.rb:17]: If I don't remove it I get this error

image

and just searching around one ai chat bot said this

Screenshot 2024-10-25 at 10 15 55

This might not be true because it's ai but I understood this to mean that this line I've removed is automatically checking and expecting the error message to be 'can't be blank' by default so I couldn't see a way of changing it. I didn't check any documentation for the 'shoulda-matchers' gem.

File: config/locales/teacher_interface.en.yml [hunk 3]
```diff
@@ -95,6 +95,9 @@ en:
           school_details_cannot_be_verified: Tell us more about this school
           unrecognised_references: Tell us more about your work history
           work_history_break: Tell us more about your work history
+    further_information_request_item_text_form:
+      response:
+        blank: Please respond to the assessor's request
```
Comment (Hassanmir92) [inline config/locales/teacher_interface.en.yml:100]: You can remove this now

**Review by Hassanmir92 — APPROVED** [review]
LGTM 💪 

Thanks for addressing comments
